src/Bot.hs

```diff
@@ -226,6 +225,18 @@ builtinCommands =
               [qms|You asked me to whisper you this: "{messageContent msg}"|]))
     ]
 
+wiggle :: Comonad w => Reaction w String
+wiggle = Reaction $ \wa -> wiggle' (extract wa) True
+  where
+    defaultTimeoutMillis = 10000
+    wiggle' name isRetry = do
+      request <- parseRequest [qms|http://localhost:8081/wiggle/{name}|]
+      response <- httpRequest request
+      let status = getResponseStatus response
+      -- TODO: wiggle retry doesn't handle the server being down
+      when (status /= ok200) $
+        when isRetry $ timeout defaultTimeoutMillis $ wiggle' name False
```

Comment:
Infinitely is not an option because it's prone to infinite loop problems.
And Implementing a circuit breaker or retry policies from the very beginning is kinda overengineered

Reply:
@tau3 makes sense. Let's keep it like that for awhile and see if it works well for us.